### DISCUSSION TITLE
Pickle detected this about the MODELS, how do I know they are safe?

### DISCUSSION THREAD

**Discussion by Stable777** [deforum-art/sd-webui-text2video#86] [Q&A]
Hello,
Pickle detected these lines of code:
![image](https://user-images.githubusercontent.com/114045211/228915201-0948329d-a09f-4bbb-ad34-49c8bb0cc854.png)

How can I be sure they are safe?
Thjx

**Comment by hithereai** [comment]
This is the wrong repo to open such issue. Please open in the modelscope repo.

**Reply by Stable777** [reply]
Hello @hithereai , sorry I am not used to Git hub much, whi is the url on my browser "https://github.com/deforum-art/sd-webui-modelscope-text2video/discussions/86", isnt that modelscope then?
Where is the right repo then?
